docs/build/voting-dapp-tutorial.md

```diff
@@ -0,0 +1,1245 @@
+---
+title: dApp Tutorial
+description: Build a complete dApp on Kadena
+---
+
+import PageRef from '@components/PageRef'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Complete dApp Tutorial
+
+Build a complete on-chain voting dApp
```

Comment:
In its most basic form, the Kadena blockchain is a digital ledger. This ledger is never stored, but rather exists on the “chain” supported by thousands of nodes simultaneously. Thanks to encryption and decentralization, the Kadena blockchain’s database of transactions is incorruptible, and each record is easily verifiable. The network cannot be taken down or influenced by a single party because it doesn’t exist in one place.

Blockchain can solve the many problems discovered in these early attempts at online voting. A blockchain-based voting application does not concern itself with the security of its Internet connection, because any hacker with access to the terminal will not be able to affect other nodes. Voters can effectively submit their vote without revealing their identity or political preferences to the public. Officials can count votes with absolute certainty, knowing that each ID can be attributed to one vote, no fakes can be created, and that tampering is impossible.

Reply:
Thanks